### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by petersem** [enchant97/hasty-paste#14] [General]
That's for your responsiveness and a great app. I've recommended it on my docker group and will be adding it as a recommended app in the group app list.

[Dockerholics ](https://www.facebook.com/groups/205764024543769/?ref=share_group_link)

**Comment by enchant97** [comment]
Thanks for your suggestions. Since publicising my apps I have really got some good feedback and ideas, and obviously users to actually use my software. My goal is to improve peoples lives by making good software.

I run a website and the content on it might be interesting for you; as I have just uploaded my first issue of "Now Self Hosted" which is reviews of other self hosted software. You can access it here: [enchantedcode.co.uk](https://enchantedcode.co.uk/posts/self-hosted/2022-08-now-self-hosted-1).

**Reply by petersem** [reply]
Thanks. Here's some more to consider. I'll be adding yours into this list soon as well. 
[Dockerholics app list](https://petersem.github.io/dockerholics/)
